include/Core/Base/Mixin/Renderer.php

```diff
@@ -84,4 +84,27 @@ protected function split_post_type_and_status( $str ) {
 
 		return $type_status;
 	}
+
+	/**
+	 * Render Tags dropdown.
+	 */
+	protected function render_user_role_dropdown() {
+		$users_count = count_users();
+		?>
+		<select name="smbd_u_roles[]" class="select2" multiple="multiple" data-placeholder="<?php _e( 'Select Role', 'bulk-delete' ); ?>">
+		<?php 
+			foreach ( $users_count['avail_roles'] as $role => $count ) {
+			$role_detail = get_role( $role );
+			if( isset( $role_detail->name ) ){
+				$role_name = $role_detail->name;
```

Comment:
You are not using the `$role_name` variable anywhere.

Is it needed? If yes, then use it. If not then remove it.